maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -75,6 +75,9 @@
 
     /**
      * @parameter  property="bundled" default-value="false"
+     *
+     * Instead of creating a `modules` folder in the distro directory, will put modules inside
```

Comment:
What would be the advantage of not creating a modules folder in the distro directory?

Reply:
I have no idea. It was implemented in https://issues.openmrs.org/browse/SDK-177 . My comment is just there to explain what it does.